front/lib/amplitude/node/index.ts

```diff
@@ -54,10 +54,8 @@ export function getBackendClient() {
 export async function trackUserMemberships(userId: ModelId) {
   const amplitude = getBackendClient();
   const user = await User.findByPk(userId);
-  const memberships = await Membership.findAll({
-    where: {
-      userId: userId,
-    },
+  const memberships = await MembershipResource.getActiveMemberships({
```

Comment:
this is a behavior change: we were tracking all user memberships each time (including revoked ones). Now we'll be tracking active ones only. cc @lasryaric